test/run_test.py

```diff
@@ -941,12 +942,12 @@ def determine_target(test, touched_files, options):
 
 
 def run_test_module(test: str, test_directory: str, options) -> Optional[str]:
-    test_module = parse_test_module(test)
+    test_module, unittest_args = parse_test(test)
 
     # Printing the date here can help diagnose which tests are slow
     print_to_stderr('Running {} ... [{}]'.format(test, datetime.now()))
-    handler = CUSTOM_HANDLERS.get(test, run_test)
-    return_code = handler(test_module, test_directory, options)
+    handler = CUSTOM_HANDLERS.get(test_module, run_test)
+    return_code = handler(test_module, test_directory, options, extra_unittest_args=unittest_args)
```

Comment:
Currently, only `run_test`, the default `handler`, accepts the `extra_unittest_args` parameter. This shouldn't be that hard to add to the others

https://github.com/pytorch/pytorch/blob/d3cde6c23c1a7d184f38d4e5fb797257e39d3376/test/run_test.py#L603-L609

Do we want that?

Reply:
Let's see if anyone requests it.